test/addons/openssl-client-cert-engine/test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+
+var engine = path.join(path.dirname(module.filename),
```

Comment:
Not exactly wrong but I'd just use `__dirname` here.

You don't even need path.join(), just `__dirname + '/build/Release/testengine.engine'` should work on all platforms, including Windows.